arex-storage-web-api/src/main/java/com/arextest/storage/service/handler/mocker/coverage/CoverageEventListener.java

```diff
@@ -10,6 +10,6 @@ public interface CoverageEventListener {
   void onBeforeNewCaseRecord(Mocker coverageMocker);
   void onNewCaseRecorded(Mocker coverageMocker);
 
-  void onBeforeExistingCaseRecord(Mocker coverageMocker);
+  boolean handleExistingCaseCondition(Mocker coverageMocker);
```

Comment:
It is recommended to rename the function to hasDuplicateCase or checkDuplicateCase, to accurately reflect its functionality.